### PR TITLE
Simplify `AcceptRedisInsightEula`

### PR DESCRIPTION
**Pull request by Alirexaa**
Follow up Redis Insight PR.

Remove Utf8JsonWriter and use JsonContent instead

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/aspire/pull/5771)

### CODE REVIEW

File: src/Aspire.Hosting.Redis/RedisBuilderExtensions.cs [hunk 1]
```diff
@@ -252,54 +253,46 @@ await pipeline.ExecuteAsync(async (ctx) =>
...
-                        var response = await client.PatchAsync(apiUrl, content, ctx)
-                        .ConfigureAwait(false);
+                    var response = await client.PatchAsync(apiUrl, content, ctx)
+                    .ConfigureAwait(false);
 
-                        response.EnsureSuccessStatusCode();
+                    response.EnsureSuccessStatusCode();
 
-                        var d = await response.Content.ReadAsStringAsync(ctx).ConfigureAwait(false);
+                    var d = await response.Content.ReadAsStringAsync(ctx).ConfigureAwait(false);
```
Comment (davidfowl) [inline src/Aspire.Hosting.Redis/RedisBuilderExtensions.cs:287]: What's this for?

Comment (Alirexaa) [reply src/Aspire.Hosting.Redis/RedisBuilderExtensions.cs]: Just debug things.
Removed.

**Review by davidfowl — APPROVED** [review]

File: src/Aspire.Hosting.Redis/RedisBuilderExtensions.cs [hunk 2]
```diff
@@ -252,54 +253,43 @@ await pipeline.ExecuteAsync(async (ctx) =>
 
         static async Task AcceptRedisInsightEula(ILogger resourceLogger, HttpClient client, CancellationToken ct)
         {
-            using (var stream = new MemoryStream())
-            {
-                using var writer = new Utf8JsonWriter(stream);
```
Comment (davidfowl) [inline src/Aspire.Hosting.Redis/RedisBuilderExtensions.cs:256]:
```suggestion
```

File: src/Aspire.Hosting.Redis/RedisBuilderExtensions.cs [hunk 3]
```diff
@@ -252,54 +253,43 @@ await pipeline.ExecuteAsync(async (ctx) =>
...
-                var pipeline = new ResiliencePipelineBuilder().AddRetry(new Polly.Retry.RetryStrategyOptions
-                {
-                    Delay = TimeSpan.FromSeconds(2),
-                    MaxRetryAttempts = 5,
-                }).Build();
+            var pipeline = new ResiliencePipelineBuilder().AddRetry(new Polly.Retry.RetryStrategyOptions
+            {
+                Delay = TimeSpan.FromSeconds(2),
+                MaxRetryAttempts = 5,
+            }).Build();
```
Comment (davidfowl) [inline src/Aspire.Hosting.Redis/RedisBuilderExtensions.cs:276]: Isn't there a way to only retry on transient http failures? This will retry on a 404 no?

**Pull request closed without merging**